Containerfile.dependencies

```diff
@@ -0,0 +1,9 @@
+FROM python:3.12
+
+RUN apt-get update
+RUN apt-get install -y r-base
+RUN apt-get install -y cmake
+RUN pip install --root-user-action=ignore -U pip
+RUN pip install --root-user-action=ignore git+https://github.com/cdcgov/pyrenew
```

Comment:
Also, why not installing a fixed version of pyrenew? Is there a release yet, @damonbayer?

Reply:
Just released [v0.1.1](https://github.com/CDCgov/PyRenew/releases/tag/v0.1.1), which we can target.

Reply:
Just added it as an argument as well.